SPECS/moby-engine/moby-engine.spec

```diff
@@ -121,12 +118,13 @@ fi
 # docker-proxy symlink in bindir to fix back-compat
 %{_bindir}/docker-proxy
 %{_libexecdir}/docker-proxy
-%dir %{_sysconfdir}/docker
```

Comment:
Checking if we should remove this directory or not. I'm worried that this directory removal could cause issues if a user has put files in here, outside of the config file that we're intentionally covering with backported change libexec for daemon.json

Reply:
Just checked the behavior. If there's something else in the folder during an rpm upgrade which removes the folder from the package, the folder itself and the user's files remain as expected in /etc/docker